modules/docs/src/main/scala/weaver/FansiModifier.scala

```diff
@@ -0,0 +1,37 @@
+package weaver
+package docs
+
+import mdoc._
+
+class FansiModifier extends mdoc.PostModifier {
```

Comment:
This is not actually used, but I struggle to part ways with it, given how ugly of a hack it is.